src/index.html

```diff
@@ -8,6 +8,13 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="body-stars">
```

Comment:
redundant

Reply:
It was not resolved. You need to remove this `<div>`